lib/dev/frame-script.js

```diff
@@ -110,6 +113,8 @@ const observer = {
 
 observerService.addObserver(observer, "content-document-interactive", false);
 observerService.addObserver(observer, "content-document-loaded", false);
-observerService.addObserver(observer, "content-page-hidden", false);
+observerService.addObserver(observer, "chrome-document-interactive", false);
+observerService.addObserver(observer, "chrome-document-loaded", false);
```

Comment:
It looks like test-dev-panel only has panels using data: URI.
It would be great if you could also test a chrome:// URL, like `chrome://global/content/about.xhtml` for ex, can we load any document or does it has to do something special to work in panel?